challenge-02/README.md

```diff
@@ -4,29 +4,43 @@ Nesse exercício, você está livre para escolher os nomes para suas variáveis
 
 ```js
 // Crie uma função que receba dois argumentos e retorne a soma dos mesmos.
-?
+function soma( x,y ){
+	return  x + y;
+}
+soma (15, 8);
+
 
 // Declare uma variável que receba a invocação da função criada acima, passando dois números quaisquer por argumento, e somando `5` ao resultado retornado da função.
-?
+
+var total = soma (15, 8)+5;
```

Comment:
-1
Ao invocar uma função, procure sempre "colar" a abertura dos parênteses no nome da função, e adicionar um espaço sempre que você for fazer uma operação, no caso da soma.